changelog/fix_style_sample_chrash.md

```diff
@@ -0,0 +1 @@
+* [#10358](https://github.com/rubocop/rubocop/pull/10358): Fix Style/Sample crash on endless range shuffle indexes. ([@gsamokovarov][])
```

Comment:
It seems that beginless range can also be mentioned :-)

Reply:
Beginless it is! Amended the commit and the PR title and description.